packages/test-helper-appid/src/router.js

```diff
@@ -0,0 +1,39 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+'use strict';
+
+var bodyParser = require('body-parser');
+var express = require('express');
+var createUser = require('./create-user');
+
+/* eslint new-cap: [0] */
+var router = module.exports = express.Router();
+
+// Enable JSON bodies
+// ------------------
```

Comment:
just a separator. when all of the test-helper-server code was in a single file, i used comments like this to break up sections. that style sort of continued when I broke it up into multiple files, so I kept it here.